sentry-android-fragment/src/main/java/io/sentry/android/fragment/SentryFragmentLifecycleCallbacks.kt

```diff
@@ -0,0 +1,91 @@
+package io.sentry.android.fragment
+
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentManager.FragmentLifecycleCallbacks
+import io.sentry.Breadcrumb
+import io.sentry.IHub
+import io.sentry.SentryLevel.INFO
+
+@Suppress("TooManyFunctions")
+internal class SentryFragmentLifecycleCallbacks(private val hub: IHub) : FragmentLifecycleCallbacks() {
```

Comment:
I've intentionally missed 

- onFragmentPreAttached
- onFragmentPreCreated
- onFragmentActivityCreated

and it's a completely subjective decision but I believe they might create too much noise. WDYT? Is it okay or should I add some configuration for the clients about which callbacks do they want to add breadcrumbs for?

Reply:
we also dont add the `Pre` breadcrumbs for the activity lifecycle, also because those methods are only available in the newer SDK versions.
`onFragmentActivityCreated` is probably only noise since the activity lifecycle will have its own, I'd keep as it is